shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/model/PaginationMetaApiModel.kt

```diff
@@ -0,0 +1,18 @@
+package co.nimblehq.blisskmmic.data.model
+
+import co.nimblehq.blisskmmic.domain.model.PaginationMeta
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+class PaginationMetaApiModel(
```

Comment:
In Android, I hardly see we use the postfix `ApiModel` for a response. Also shouldn't we use `data class` here instead? 🤔
```suggestion
data class PaginationMetaResponse(
```

Reply:
About the `ApiModel` suffix, it follows the Android doc: https://developer.android.com/topic/architecture/data-layer#business-models

Reply:
Agree with `xxxResponse`

Reply:
@suho FYI, our Android team never uses it as far as I recall in existing projects and we use `Response` mostly. But if it is from the official Android so I am fine both way, nice to know 🙏

Reply:
I'm picking `ApiModel` to match Android document 🙏